charger/openevse.go

```diff
@@ -155,17 +155,17 @@ func (c *OpenEVSE) PerformRAPICommand(uri, command string) (response string, suc
 	uriBuilder.WriteString(uri)
```

Comment:
Is this a JSON request or something else? If json use `GetJSON`?

Reply:
It’s plain text.

Reply:
You could use the helper's `GetBody` then. It conveniently checks for HTTP 200 already

Reply:
Thanks. I'm working on it. 
I was actually wrong and it's JSON, sorry.

Reply:
No problem. I'm a bit a sucker for "nice" code ;)